administrator/templates/atum/scss/blocks/_quickicons.scss

```diff
@@ -229,16 +271,17 @@
 
 #wrapper.closed {
   .quick-icons {
-    li {
+    .single,
+    .quickicon-group {
       @media (max-width: 1180px) {
         width: calc(50% - 1rem);
         max-width: calc(50% - 1rem);
         flex: 1 1 calc(50% - 1rem);
       }
       @media (max-width: 860px) {
-        width: calc(100% - 1rem);
-        max-width: calc(100% - 1rem);
-        flex: 1 1 calc(100% - 1rem);
+        width: 100%;
```

Comment:
do you really have to set them all to 100% ??

I dont' know about flex but I was taught
if width > max-width; let the browser use max-width.
if width < max-width; let the browser use width.

Reply:
Fixed. With is here 100% per default,